R/utils.R

```diff
@@ -1,61 +1,59 @@
-#'   rotate by default (`FALSE`).
+#' rotate by default (`FALSE`).
 #' @param ggtheme optional, (`character`) `ggplot2` theme to be used by default. Defaults to `"gray"`.
 #' @param ggplot2_args (`ggplot2_args`) object created by [teal.widgets::ggplot2_args()]
-#'  with settings for the module plot.
-#'  The argument is merged with options variable `teal.ggplot2_args` and default module setup.
+#' with settings for the module plot.
+#' The argument is merged with options variable `teal.ggplot2_args` and default module setup.
 #'
-#'  For more details see the vignette: `vignette("custom-ggplot2-arguments", package = "teal.widgets")`
+#' For more details see the vignette: `vignette("custom-ggplot2-arguments", package = "teal.widgets")`
 #' @param basic_table_args (`basic_table_args`) object created by [teal.widgets::basic_table_args()]
-#'  with settings for the module table.
-#'  The argument is merged with options variable `teal.basic_table_args` and default module setup.
+#' with settings for the module table.
+#' The argument is merged with options variable `teal.basic_table_args` and default module setup.
 #'
-#'  For more details see the vignette: `vignette("custom-basic-table-arguments", package = "teal.widgets")`
-#' @param pre_output (`shiny.tag`, optional)\cr
+#' For more details see the vignette: `vignette("custom-basic-table-arguments", package = "teal.widgets")`
+#' @param pre_output (`shiny.tag`, optional) Text or UI element to be displayed before the module's output,
+#' providing context or a title.
 #'  with text placed before the output to put the output into context. For example a title.
-#' @param post_output (`shiny.tag`, optional) with text placed after the output to put the output
-#' into context. For example the [shiny::helpText()] elements are useful.
+#' @param post_output (`shiny.tag`, optional) Text or UI element to be displayed after the module's output,
+#' adding context or further instructions. Elements like `shiny::helpText()` are useful.
 #'
 #' @return Object of class `teal_module` to be used in `teal` applications.
 #'
 #' @name shared_params
 #' @keywords internal
 NULL
 
-#' Add axis labels that show facetting variable
+#' Add labels for facets to a ggplot2 object
 #'
-#' Add axis labels that show facetting variable
+#' Enhances a ggplot2 plot by adding labels that describe
+#' the faceting variables along the x and y axes.
 #'
-#' @param p `ggplot2` object to add facet labels to
-#' @param xfacet_label label of facet along x axis (nothing created if NULL),
-#'   if vector, will be concatenated with " & "
-#' @param yfacet_label label of facet along y axis (nothing created if NULL),
-#'   if vector, will be concatenated with " & "
+#' @param p (`ggplot2`) object to which facet labels will be added.
+#' @param xfacet_label (`character`) Label for the facet along the x-axis.
+#' If NULL, no label is added. If a vector, labels are joined with " & ".
```

Comment:
Can we add NULL with backticks`NULL`? I see some of the previous docs followed this.